source/Halibut/Transport/Streams/TimeoutStreamWrapper.cs

```diff
@@ -0,0 +1,192 @@
+﻿#nullable enable
+using System;
+using System.IO;
+using System.Runtime.Remoting;
+using System.Threading;
+using System.Threading.Tasks;
+using Halibut.Util;
+
+namespace Halibut.Transport.Streams
+{
+    class TimeoutStreamWrapper : Stream
```

Comment:
TimeoutStreamWrapper or just TimeoutStream?

Just wondering if the word 'Wrapper' is redundant (as streams often wrap other streams)

Reply:
Changed to TimeoutStream